WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -619,6 +619,10 @@ private void removeDotComCredentials() {
         }
     }
 
+    public int getAllBlogsCount() {
+        return  getAllBlogs().size();
```

Comment:
Retrieving all the blogs simply to get their count isn't optimal. Can we rely on [one of these existing routines](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/delete-site-delete-site/WordPress/src/main/java/org/wordpress/android/WordPressDB.java#L588-L598) instead?